libvirt/resource_libvirt_domain_test.go

```diff
@@ -540,3 +542,91 @@ func testAccCheckLibvirtScsiDisk(n string, domain *libvirt.Domain) resource.Test
 		return nil
 	}
 }
+
+func TestAccLibvirtDomain_FirmwareNoTemplate(t *testing.T) {
+	// var nvram_path = "/var/lib/libvirt/qemu/nvram/terraform-test-firmware-no-template_VARS.fd"
+	var nvram_path = "/tmp/terraform-test-firmware-no-template_VARS.fd"
+	var domain libvirt.Domain
+	var config = fmt.Sprintf(`
+            resource "libvirt_domain" "acceptance-test-domain" {
+                    name = "terraform-test-firmware-no-template"
+                    nvram = "%s"
+  	                firmware = "/usr/share/ovmf/OVMF.fd"
+            }`, nvram_path)
+	// size of an accepted, valid, nvram backing store
+	nvram_dummy := make([]byte, 131072)
+	err := ioutil.WriteFile(nvram_path, nvram_dummy, 0777)
```

Comment:
Can you refactor this code to use [io/ioutil/TempFile](https://golang.org/pkg/io/ioutil/#TempFile)?